From 121c5a98385c815481d6c71654db8d3a606d6345 Mon Sep 17 00:00:00 2001
From: Laurent FERT <laurent.fert@intel.com>
Date: Thu, 20 Jun 2013 14:41:59 +0200
Subject: [PORT FROM MAIN][BT] Add bt_vendor.conf for byt_t and byt_m

BZ: 117525

Add libbt configuration file bt_vendor.conf for byt_t_ffrd8, byt_t_ffrd10 and
byt_m_crb variants.

Change-Id: I9b855a58892d7b51f733aa7ced307d366c27f2cf
Signed-off-by: Laurent FERT <laurent.fert@intel.com>
Signed-off-by: Patrick Combes <patrick.combes@intel.com>
Reviewed-on: http://android.intel.com:8080/114601
Reviewed-by: cactus <cactus@intel.com>
Reviewed-by: Boniface, XavierX <xavierx.boniface@intel.com>
Reviewed-by: Gros, Julien <julien.gros@intel.com>
Reviewed-by: Aquilone, RaffaeleX <raffaelex.aquilone@intel.com>
Reviewed-by: Molero Rodenas, BeatrizX <beatrizx.molero.rodenas@intel.com>
Reviewed-by: Tinelli, Vincent <vincent.tinelli@intel.com>
Tested-by: Tinelli, Vincent <vincent.tinelli@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
---
 libbt/conf/intel/byt_m_crb/bt_vendor.conf    |    5 +++++
 libbt/conf/intel/byt_t_ffrd10/bt_vendor.conf |    5 +++++
 libbt/conf/intel/byt_t_ffrd8/bt_vendor.conf  |    5 +++++
 3 files changed, 15 insertions(+), 0 deletions(-)
 create mode 100644 libbt/conf/intel/byt_m_crb/bt_vendor.conf
 create mode 100644 libbt/conf/intel/byt_t_ffrd10/bt_vendor.conf
 create mode 100644 libbt/conf/intel/byt_t_ffrd8/bt_vendor.conf

diff --git a/libbt/conf/intel/byt_m_crb/bt_vendor.conf b/libbt/conf/intel/byt_m_crb/bt_vendor.conf
new file mode 100644
index 0000000..93aa23c
--- /dev/null
+++ b/libbt/conf/intel/byt_m_crb/bt_vendor.conf
@@ -0,0 +1,5 @@
+# UART device port where Bluetooth controller is attached
+UartPort = /dev/ttyMFD0
+
+# Firmware patch file location
+FwPatchFileRootPath = /etc/firmware/bt/
diff --git a/libbt/conf/intel/byt_t_ffrd10/bt_vendor.conf b/libbt/conf/intel/byt_t_ffrd10/bt_vendor.conf
new file mode 100644
index 0000000..93aa23c
--- /dev/null
+++ b/libbt/conf/intel/byt_t_ffrd10/bt_vendor.conf
@@ -0,0 +1,5 @@
+# UART device port where Bluetooth controller is attached
+UartPort = /dev/ttyMFD0
+
+# Firmware patch file location
+FwPatchFileRootPath = /etc/firmware/bt/
diff --git a/libbt/conf/intel/byt_t_ffrd8/bt_vendor.conf b/libbt/conf/intel/byt_t_ffrd8/bt_vendor.conf
new file mode 100644
index 0000000..93aa23c
--- /dev/null
+++ b/libbt/conf/intel/byt_t_ffrd8/bt_vendor.conf
@@ -0,0 +1,5 @@
+# UART device port where Bluetooth controller is attached
+UartPort = /dev/ttyMFD0
+
+# Firmware patch file location
+FwPatchFileRootPath = /etc/firmware/bt/
-- 
1.7.4.1

